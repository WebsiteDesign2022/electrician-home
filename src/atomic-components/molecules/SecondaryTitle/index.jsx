import React from "react"

import TopLeftTriangle from "../../atoms/TopLeftTriangle";

const SecondaryTitle = ({ titlePartOne, titlePartTwo }) => (<div>
    <TopLeftTriangle color="black"/>
    <h1 className="display-5 lh-1 mb-3 justify-content-end" align="left">{titlePartOne}</h1>
    <h1 className="display-5 fw-bold lh-1 mb-3 align-content-start justify-content-end" align="left">{titlePartTwo}</h1>
</div>);

export default SecondaryTitle;