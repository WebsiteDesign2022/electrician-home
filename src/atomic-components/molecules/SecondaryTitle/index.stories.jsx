import React from 'react';

import SecondaryTitle from '.';

export default {
  title: 'Electrician-Home/molecules/SecondaryTitle',
  component: SecondaryTitle,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <SecondaryTitle {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {
    titlePartOne: 'Title Part One', 
    titlePartTwo: 'Title Part Two'
};
