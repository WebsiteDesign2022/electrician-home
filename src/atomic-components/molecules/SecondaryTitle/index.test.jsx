import { render } from '@testing-library/react'

import SecondaryTitle from '.';

describe("SecondaryTitle", () => {
    test("renders SecondaryTitle and matches snapshot", () => {
        render(
            <SecondaryTitle
                titlePartOne="title part one" 
                titlePartTwo="title part two"
            />
        );

        expect(screen).toMatchSnapshot();

    });
});
