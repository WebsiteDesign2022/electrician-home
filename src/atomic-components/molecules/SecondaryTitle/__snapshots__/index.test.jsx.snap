// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryTitle renders SecondaryTitle and matches snapshot 1`] = `Screen {}`;
